lib/components/admin/field-trip-details.js

```diff
@@ -214,7 +223,8 @@ const mapStateToProps = (state, ownProps) => {
     currentQuery: state.otp.currentQuery,
     datastoreUrl: state.otp.config.datastoreUrl,
     dateFormat: getDateFormat(state.otp.config),
-    request
+    request,
+    session: state.callTaker.session
```

Comment:
Nit: `session` could be `sessionId` instead since `session` isn't used except to get the `sessionId` field.

Reply:
Changed in https://github.com/opentripplanner/otp-react-redux/pull/398/commits/6ef72b9b8d00366f5c62717405be677c0bb34233.